pkg/controller/argocd/util.go

```diff
@@ -88,6 +89,21 @@ func getArgoApplicationControllerResources(cr *argoprojv1a1.ArgoCD) corev1.Resou
 	return resources
 }
 
+// getArgoApplicationControllerCommand will return the command for the ArgoCD Application Controller component.
+func getArgoApplicationControllerCommand(cr *argoprojv1a1.ArgoCD) []string {
+	cmd := []string{
+		"argocd-application-controller",
+		"--operation-processors", fmt.Sprint(getArgoServerOperationProcessors(cr)),
+		"--redis", getRedisServerAddress(cr),
+		"--repo-server", nameWithSuffix("repo-server:8081", cr),
```

Comment:
Please make sure you pick up #263 to get fqdn for server hostnames.

Reply:
done !!